recipes/libzip/all/conanfile.py

```diff
@@ -115,7 +115,7 @@ def _patch_sources(self):
         # Honor zstd enabled
         if self._has_zstd_support:
             lib_cmakelists = os.path.join(self.source_folder, "lib", "CMakeLists.txt")
-            replace_in_file(self, top_cmakelists, "find_package(Zstd)", "find_package(zstd)")
+            replace_in_file(self, top_cmakelists, "find_package(Zstd", "find_package(zstd")
```

Comment:
@prince-chrismc, I guess we should read the version they pass (if it exists) and adjust our requirements accordingly. This would be the right solution. There could be another, more pragmatic one since Conan.tools API is limited and we don't want to make complicated recipes, do we?

Reply:
Maybe

The match part is an if statement `find_package (zstd 1.3.6) if Version(self.version) >= 1.9.2 else find_package(ztd)`

Type of thing?

Reply:
Vacation wifi won't let me edit and put in the double quotes for the strings 🙃

Reply:
Maybe a map for future releases? But yes, one above is easy

Reply:
One big problem I see - is that recipe is not easily updatable once this version changes... but let's be pragmatic and start with something... I'll wrap it into a function so we can change this anytime

Reply:
Yeah it's painful no matter what